### PR TITLE
Convert zip#53

### PR DESCRIPTION
**Pull request by asmolentzov**
When a user creates a garden and enters a zip code, the app can now makes a call to the Google Geocoding API to return the latitude and longitude associated with that zip code (to use for future calls to a weather API). 
- Adds lat and long attributes to Garden model
- Adds set_lat_long before_save callback method to Garden that sets the lat and the long
- Adds LocationService that makes the call to the Google Geocoding API
- Adds Zipcode model that stores a zip code and its associated latitude and longitude in the database
- Adds ZipcodeFinder PORO that either finds the zip code if it exists in the DB or makes a call to get the lat and long and then saves a new Zipcode to the DB. 
- Locally: All tests passing
- Travis: Tests NOT passing - need to add environment variable for google geocoding api key. Looked and not sure if @le3ah has to do it since it's your repo?

### CODE REVIEW

**Comment by bendelonlee** [comment]
Fantastic progress!!  I'll review it all when I get a chance.
Would it be better to add VCR at this point instead of giving our keys to Travis?

**Comment by asmolentzov** [comment]
Good call, now is def a good time to set up VCR!

File: app/services/location_service.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class LocationService
+  def initialize(zip_code)
+    @zip_code = zip_code
+  end
+
```
Comment (abenetka) [inline app/services/location_service.rb]: Should we add a private to this service?

Comment (asmolentzov) [reply app/services/location_service.rb]: Good call! I'll add that before get_location_data.

**Comment by bendelonlee** [comment]
Glad you figured out what the bug was, I was having a hard time.

**Pull request merged**